src/format.iced

```diff
@@ -0,0 +1,122 @@
+stream = require('keybase-chunk-stream')
+util = require('./util')
+
+space = Buffer.from(' ')
+newline = Buffer.from('\n')
+punctuation = Buffer.from('.')
+
+words_per_line = 200
+chars_per_word = 15
+
+noop = () ->
+
+exports.FormatStream = class FormatStream extends stream.ChunkStream
+
+  __format : (chunk) ->
+    return Buffer.concat([space, chunk])
+
+  _format : (chunk) ->
+    if chunk.length < chars_per_word then return chunk
+    results = []
+    for i in [0...chunk.length] by chars_per_word
+      word = chunk[i...i+chars_per_word]
+      if i+chars_per_word >= chunk.length
+        results.push(word)
+      else
+        if @_word_count % 200 is 0 and @_word_count isnt 0 then word = Buffer.concat([word, newline])
+        else word = Buffer.concat([word, space])
+        ++@_word_count
+        results.push(word)
+    return Buffer.concat(results)
+
+  _transform : (chunk, encoding, cb) ->
+    if not @_header_written
+      @push(Buffer.concat([@_header, punctuation, space]))
+      @_header_written = true
+    super(chunk, encoding, cb)
+
+  _flush : (cb) ->
+    super(noop)
+    @push(Buffer.concat([punctuation, space, @_footer, punctuation]))
+    cb()
+
+  constructor : (opts) ->
+    if opts?.brand? then _brand = opts.brand else _brand = 'KEYBASE'
+    @_header = Buffer.from("BEGIN#{space}#{_brand}#{space}SALTPACK#{space}ENCRYPTED#{space}MESSAGE")
+    @_footer = Buffer.from("END#{space}#{_brand}#{space}SALTPACK#{space}ENCRYPTED#{space}MESSAGE")
```

Comment:
If we decide to change the delimiter in the future, all we have to do is change the global variable at the top of the file.